cli/flox/doc/manifest.toml.md

```diff
@@ -616,10 +616,12 @@ See `flox-build(1)` for more details.
 The full set of options is shown below:
 ```
 BuildDescriptor ::= {
-  command     = STRING
-, sandbox     = null | ("off" | "pure")
-, version     = STRING | VersionFile | VersionCommand
-, description = null | STRING
+  command          = STRING
+, sandbox          = null | ("off" | "pure")
+, version          = null | STRING | VersionFile | VersionCommand
+, description      = null | STRING
+, runtime-packages = null | [<STRING>, ...]
+, license          = null | [<STRING>, ...]
```

Comment:
Did we mean to make this singular when it's a list?

Reply:
I don't understand the question.

Reply:
I seem to recall there was verbal agreement yesterday to rename. Are we renaming all the way through before merging?